client/src/components/GithubUserLink.tsx

```diff
@@ -15,7 +16,7 @@ export function GithubUserLink({ value, isUserIconHidden = false }: Props) {
             <GithubAvatar githubId={value} size={24} />{' '}
           </>
         )}
-        {value}
+        {fullName ?? value}
```

Comment:
Minor, but I think this is one of the cases where `||` is better than `??` as it's better to treat empty string as missing value
```suggestion
        {fullName || value}
```